build.gradle

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id "com.github.spotbugs" version "4.0.5"
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "de.undercouch.download" version "4.0.4"
+    id "net.researchgate.release" version "2.8.0"
+}
+
+ext.ballerinaLangVersion = project.ballerinaLangVersion
+ext.puppycrawlCheckstyleVersion = "8.18"
+ext.testngVersion = "6.14.3"
+ext.slf4jVersion = "1.7.30"
```

Comment:
We don't need `ext.version`s. Instead directly use the versions mentioned in the `gradle.properties` file.

The testng, checkstyle, and slf4j version too should be defined in the gradle.properties file.

The versions mentioned in the plugins section too can be defined the in gradle.properties file. This helps to keep all the version numbers in a single place.

Reply:
Removed these `ext.version` s in ffcf734 since they were unnecessary.